docs/dev_guide/code_standards.rst

```diff
@@ -52,24 +52,102 @@ Coding Style/Conventions
 Formatting
 ==========
 
-We enforce a minimum level of code style and we have tools that will automate this step for you.
+We enforce a minimum level of code style with our continuous intergration (the name is ``sunpy.sunpy (python_codestyle [linux]``).
```

Comment:
The new stuff in the brackets makes no sense to me.

Reply:
You mean the name?

Reply:
Yeah, what does "the name is..." mean?

Reply:
I will fix that.

Reply:
See if you think it is clearer now.